index.html

```diff
@@ -5330,49 +5329,88 @@ <h2>Service Privacy</h2>
-      </p>
-      <ol>
-          <li>
-Intentional public disclosures
-          </li>
-          <li>
-<a>DIDs</a> about natural persons (as might be covered by GDPR and similar
-privacy regulations)
-          </li>
-          <li>
-<a>DIDs</a> about things and documents that are associated with natural persons
-          </li>
-      </ol>
-      <p>
-For the first category, consider non-DID publication mechanisms with only
-a single <a>service endpoint</a>. In some cases, the publication mechanism might
-reference a <a>DID Document</a> with no <a>service endpoints</a> at all. For the second
-category, prefer using only one <a>service</a> that points to an authorization server,
-mediator, or proxy that can provide herd privacy. For the third category,
-avoid the use of multiple <a>service endpoints</a> for a <a>DID</a> because some
-of these (e.g., an authorization server) are likely to be reused with other,
-related <a>DIDs</a>. Place correlatable <a>service endpoints</a> behind a
-privacy-preserving  mechanism, if possible, or introduce a mediator or proxy
-as a sole <a>service endpoint</a> in order to obscure related devices and documents
-through herd privacy.
+The degree of additional privacy risk caused by using multiple <a>service
+endpoints</a> in one <a>DID document</a> can be difficult to estimate. Privacy
+harms are typically unintended consequences. <a>DIDs</a> can identify documents,
+<a>services</a>, schemas, and other things that may be associated
+with individual people, households, clubs, and employers &mdash; and
+correlation of their <a>service endpoints</a> could become a powerful
+surveillance and inference tool.
       </p>
+<p>
+The variety of possible endpoints makes it particularly challenging to maintain 
+herd privacy, in which no information about the DID Subject is leaked by features 
+in DID Core.
+</p>
+<p>
+<strong>First</strong>, because service endpoints may be specified as URIs, they could 
+unintentionally leak personal information because of the architecture of the service. 
+For example, a service endpoint of “http://example.com/MyFirstName” is leaking the term 
+“MyFirstName” to everyone who can access the DID Document. When linking to legacy systems, 
+this is an unavoidable risk and care should be taken in such cases. We encourage new, 
+DID-aware endpoints to use nothing more than the DID itself for any identification necessary,
+e.g., “http://example.com/did%3Aexample%3Aabc123”. Since that “did:example:abc123” is already 
+exposed in the DID Document, it leaks no additional information.
+</p>
+
+<p>
+<strong>Second</strong>, because a DID Document may list multiple service endpoints, it is possible to 
+irreversibly associate services that are not associated in any other context. This 
+correlation on its own may lead to de-anonymization, revealing information about the 
+DID Subject even if the URIs used did not.
```

Comment:
```suggestion
Second, because a <a>DID document</a> can list multiple service endpoints, it is possible to 
irreversibly associate services that are not associated in any other context. This 
correlation on its own may lead to de-anonymization, revealing information about the 
<a>DID subject</a> even if the <a>URIs</a> used did not.
```